packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -40,6 +40,7 @@ type InsertOptions = ParameterOptions &
 
 type BulkInsertOptions = ParameterOptions & {
   hasTrigger?: boolean;
+  bindParam?: false | ((value: unknown) => string);
```

Comment:
`bindParam` is a legacy option. Could you add support for `parameterStyle` instead?

```ts
enum ParameterStyle {
  bind = 'bind',
  replacement = 'replacement',
}
```